yarn.lock

```diff
@@ -108,9 +108,9 @@
     "@babel/helper-plugin-utils" "^7.14.5"
```

Comment:
What is the policy regarding `yarn.lock` and `package-lock.json`? For my "forgot password" task I've updated `package-lock.json` but not `yarn.lock`, while you have done the opposite here ... which one is leading, or are both leading and should we keep both up to date? It is rather confusing that we have 2 different lock files ...

Reply:
we default to `yarn`, I think we only have `yarn` cmds in all the README's :)

so we should be deleting `package-lock.json` whenever we see it